core/src/lib/code-generation/custom-snippet.ts

```diff
@@ -93,7 +90,45 @@ export function initCustomSnippet(): CustomSnippets {
             element.getterFunction = newGetterFunction;
         }
     });
-    return rules;
+    defaultCustomSnippets.addForFillForm({
```

Comment:
Musst du nicht aber hätte zur Übersichtlichkeit addForGetterFunctions und addForFillForm in eigene Funktionen verlegt

Reply:
The Idea is to create an library for custom snippets, this code will disappear from this file. I think right now it is not to much.